src/include/migraphx/stringutils.hpp

```diff
@@ -95,6 +95,14 @@ inline std::vector<std::string> split_string(const std::string& s, char delim)
     return elems;
 }
 
+template <class LeftF, class RightF>
+std::string trim(const std::string& s, LeftF lf, RightF rf)
```

Comment:
add test inside `test/stringutils.cpp`.

Reply:
We don't actually test `stringutils.trim()` it seems.

Reply:
Can we add some unit-tests then ?